### DISCUSSION TITLE
Import dashboard API

### DISCUSSION THREAD

**Discussion by Davidkramer1999** [apache/superset#27381] [General]
### Bug description

So I have written the script which enables you to migrate trought api dashboards. So this code worked prfectly worked on version 2.0. I'm keep getting error when calling "POST /api/v1/dashboard/import/ HTTP/1.1".  This is the code i have used. Pyhton version 3.9
https://github.com/Davidkramer1999/superset_migration

### How to reproduce the bug

1. Call python3.9 app.py 
2. See to logs it will return  " 400 Bad Request: The CSRF token is missing"

### Screenshots/recordings

From docker logs 
![image](https://github.com/apache/superset/assets/74869455/25c566dc-4cec-42ed-8366-ceff8df02203)

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by Ayoub-BOUCHACHIA** [comment]
Hello, 

 use session.post instead of requests.post
